### PR TITLE
chore: add test for recursive text splitting

### PR DESCRIPTION
**Pull request by Askir**

### CODE REVIEW

File: projects/pgai/tests/vectorizer/conftest.py [hunk 1]
```diff
@@ -42,6 +51,7 @@ def vcr_():
         record_mode=vcr.mode.ONCE,
         filter_headers=["authorization"],
         match_on=["method", "scheme", "host", "port", "path", "query", "body"],
+        before_record_response=remove_set_cookie_header,
```
[the remark this answers is not in the document]
Comment (alejandrodnm) [reply projects/pgai/tests/vectorizer/conftest.py:54]: Does adding it to the `filter_header` list achieves the same?

[the remark this answers is not in the document]
Comment (Askir) [reply projects/pgai/tests/vectorizer/conftest.py:54]: yes, the filter_header only filters request headers not response header sadly.

**Review by alejandrodnm — APPROVED** [review]

File: docs/python-integration.md
Comment (smoya) [inline docs/python-integration.md:1]: I guess this file shouldn't be here.

**Review by smoya — CHANGES_REQUESTED** [review]
You are adding the `docs/python-integration.md` here (empty btw). I guess you want to remove it.

File: docs/python-integration.md
Comment (Askir) [reply docs/python-integration.md:1]: Yes not sure how this happened, I also thought this was already merged. My bad 😅

**Review by smoya — APPROVED** [review]
LGTM 🚀

**Pull request merged**